src/m365/spo/commands/page/page-section-add.ts

```diff
@@ -101,6 +166,40 @@ class SpoPageSectionAddCommand extends SpoCommand {
           }
         }
 
+        if (typeof args.options.iconAlignment !== 'undefined') {
+          if (SpoPageSectionAddCommand.IconAlignment.indexOf(args.options.iconAlignment) === -1) {
+            return 'The value of parameter iconAlignment must be Left|Right';
+          }
+        }
+
+        if (typeof args.options.fillMode !== 'undefined') {
+          if (SpoPageSectionAddCommand.FillMode.indexOf(args.options.fillMode) === -1) {
+            return 'The value of parameter fillMode must be ScaleToFill|ScaleToFit|Tile|OriginalSize';
+          }
+        }
+
+        if (args.options.zoneEmphasis !== 'Image' && (args.options.imageUrl || args.options.imageWidth ||
```

Comment:
Make sure zoneEmphasis can be case-insensitive.

Reply:
Done